app.js

```diff
@@ -233,6 +233,31 @@ $("menu_top_right_contents").addEventListener("click", event => {
 });
 
 //
+// UI events to set the font size
+//
+$("decrease_font_size").addEventListener("click", event => {
+  var font_size_input = $("input_fontsize");
+  if (font_size_input.value > 0) {
+    font_size_input.value = parseInt(font_size_input.value)-1;
+  }
+  bookViewer.fontSize = font_size_input.value + "pt";
+  console.log("Updating font size " + font_size_input.value);
+});
+
+$("increase_font_size").addEventListener("click", event => {
+  var font_size_input = $("input_fontsize");
+  font_size_input.value = parseInt(font_size_input.value)+1;
+  bookViewer.fontSize = font_size_input.value + "pt";
+  console.log("Updating font size " + font_size_input.value);
+});
+
+$("input_fontsize").addEventListener("change", event => {
```

Comment:
I come to realize that this should probably be just a label, rather than an `<input>`. Nobody is going to write a number.